front_end/panels/react_devtools/ReactDevToolsModel.ts

```diff
@@ -163,10 +163,21 @@ export class ReactDevToolsModel extends SDK.SDKModel.SDKModel<EventTypes> {
 
   #finishInitializationAndNotify(): void {
     this.#bridge = ReactDevTools.createBridge(this.#wall);
-    this.#store = ReactDevTools.createStore(this.#bridge);
+    this.#store = ReactDevTools.createStore(this.#bridge, {
+      supportsReloadAndProfile: true,
+    });
+    this.#attachReloadToProfileListener();
     this.dispatchEventToListeners(Events.InitializationCompleted);
   }
 
+  #attachReloadToProfileListener(): void {
+    this.#wall.listen((message: ReactDevToolsTypes.Message): void => {
+      if (message.event === 'reloadAppForProfiling') {
+        SDK.ResourceTreeModel.ResourceTreeModel.reloadAllPages(false);
+      }
+    });
```

Comment:
Use `bridge` for it, not `wall`. Ideally, we should be using `Store`, because this should be the only public API entrypoint.

But please use `bridge`, just to make sure we use same thing as what extension is using, in case we are going to change this.